README.md

```diff
@@ -1,3 +1,25 @@
-# myDiary
-myDairy is an online journal that allows users to make daily journal entries.
-The site is published at https://sammuel09.github.io/myDiary/
+# Project Title
+myDiary is a web-based journal application. 
+
+## Getting Started
+In order to have a copy of this project, clone this project to your local repository.
+
+### Prerequisites
+You need to have NodeJs installed on yourlocal machine. Install Node [here](https://nodejs.org/en/)
+
+### Frameworks Used
+ExpressJs. Download as an npm package [here](https://expressjs.com/)
+
+### Coding Style
+This project was built using Javascript ES6. So, to transpile the ES6 code to ES5, you need to install [BabelJS](https://babeljs.io/) an NPM package. 
+
+## Testing
+To test, you need to install [mocha](https://mochajs.org/), [chai](http://www.chaijs.com/) and [chai-http](http://www.chaijs.com/plugins/chai-http/). You also need to install [babel-register](https://www.npmjs.com/package/@babel/register) as NPM package to transpile ES6 code on the fly for testing.
+
+###
+Run the test with the NPM command: ```npm test```
+
+## Hosted
```

Comment:
`Hosted` is definitely not the appropriate title for this section

Reply:
OK. Sir.